src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -53,7 +53,8 @@ HighlightingPage::HighlightingPage()
         // getSettings()->enableHighlights));
 
         // TABS
-        auto tabs = layout.emplace<QTabWidget>();
+        this->tabWidget_ = new QTabWidget();
+        auto tabs = layout.append(tabWidget_);
```

Comment:
```suggestion
        auto tabs = layout.emplace<QTabWidget>();
        this->tabWidget_ = tabs;
```

Reply:
This does not work. Using emplace creates a `LayoutCreator<QTabWidget>` which was exactly the reason for using append in this instance.
You can do `this->tabWidget_ = &*tabs;` but for readability I prefer the previous version.